packages/datepickers/src/elements/Datepicker/Datepicker.tsx

```diff
@@ -97,6 +97,9 @@ export interface IDatepickerProps {
   zIndex?: number;
 }
 
+/**
+ * @extends HTMLAttributes<HTMLElement>
+ */
```

Comment:
I think we remove this one as there is no ref being passed or HTML props being spread to the component.

Reply:
Ahh okay, makes sense